common/src/main/java/org/candlepin/common/exceptions/ExceptionMessage.java

```diff
@@ -25,7 +27,7 @@
  */
 @XmlRootElement
 @XmlAccessorType(XmlAccessType.FIELD)
-public class ExceptionMessage {
+public class ExceptionMessage implements Serializable {
```

Comment:
Needs a serialVersionUUID.  There was some trouble a few months back due to classes implementing Serializable that lacked a version.